README.md

```diff
@@ -0,0 +1,6 @@
+### LaunchDarkly Sample Go Application  ###
+We've built a simple console application that demonstrates how LaunchDarkly's SDK works.  Below, you'll find the basic build procedure, but for more comprehensive instructions, you can visit your [Quickstart page](https://app.launchdarkly.com/quickstart#/).
+##### Build instructions  #####
+0. Get and install [Godep](https://github.com/tools/godep)
```

Comment:
Better would be: 

"Make sure you have [godep](https://github.com/tools/godep) installed: `go get github.com/tools/godep`